views/prisonerProfile/prisonerIncentiveLevelDetails.njk

```diff
@@ -42,15 +42,15 @@
     <div class="horizontal-information govuk-!-margin-bottom-3">
         <div class="horizontal-information__item">
             <h3 class="govuk-heading-s govuk-!-margin-bottom-1">Current incentive level</h3>
-            <p class="govuk-body" data-test="incident-time">{{ currentIepLevel }}</p>
+            <p class="govuk-body" data-test="current-iep">{{ currentIepLevel }}</p>
         </div>
         <div class="horizontal-information__item">
             <h3 class="govuk-heading-s govuk-!-margin-bottom-1">Time since review</h3>
-            <p class="govuk-body" data-test="incident-location">{{ daysOnIepLevel }}</p>
+            <p class="govuk-body" data-test="days-since-review">{{ daysOnIepLevel }}</p>
```

Comment:
I know they may often be the same but `data-test` attribute suggests this "days since last review" but the value name suggests is "days on level" 🤔

Assuming the value is correct maybe change to `data-test="days-on-level"`?

Reply:
the only point of this commit was to highlight that the existing label was meaningless (and unused). in a later commit, i change it to something different

Reply:
Oh I see, sorry I was reviewing this commit-by-commit and I commented on something that changed. I see these labels are now correct :)